### DISCUSSION TITLE
File Exists error during build

### DISCUSSION THREAD

**Discussion by DuncanLHS** [danny-avila/LibreChat#2076] [Troubleshooting]
### What happened?

Error during docker compose, attempting to duplicate a folder.

I've manually removed all folders created in the project root and re-run but error still occurs. Looks like there's duplicate mkdir commands being run during the build process?

Windows 11

### Steps to Reproduce

1. Git pull repo
2. Run `docker compose up --build`

### Relevant log output

```shell
Error response from daemon: error while creating mount source path '[repo location]/meili_data_v1.6': mkdir [repo location]/meili_data_v1.6: file exists
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Just delete the `/meili_data_v1.6` directory, next startup may be slow due to indexing but no issue with this at all

**Reply by DuncanLHS** [reply]
No, that doesn't fix the issue. As above, I deleted all the folders created during startup.
